### PR TITLE
Avoid missing teams.

### LINKED ISSUE #262: Avoid missing teams.
Some leaders only select `团体赛` for athletes but forget to fill the relays.

### PR DESCRIPTION
**Pull request by JeldorPKU**
Fix #262 .

### CODE REVIEW

File: application/controllers/registration.php [hunk 1]
```diff
@@ -245,6 +245,8 @@ public function team() {
             $key_set = array();
             if (!$team_post) $team_post = array();
             if (sizeof($team_post) > 3) exit(err_custom_msg('2002', array()));
+            $relay_athletes_number = $this->people->get_relay_athletes_number($school_id);
```
Comment (huxuan) [inline application/controllers/registration.php]: I think you mean `real` or `really`, right?

Comment (JeldorPKU) [inline application/controllers/registration.php]: **relay** has the meaning 接力赛, which I have just found in the dictionary.

Comment (huxuan) [inline application/controllers/registration.php]: OK, that convince me well.

Comment (huxuan) [inline application/controllers/registration.php]: Then what about use `team` instead?

Comment (JeldorPKU) [inline application/controllers/registration.php]: Good idea for variable consistency. Will do it soon.

**Pull request merged**